ha/k8s_setup/ha_setup.py

```diff
@@ -196,32 +221,38 @@ def process(self):
             machine_id = Conf.machine_id
             ha_log_path = os.path.join(log_path, f'ha/{machine_id}')
 
-            consul_endpoints = Conf.get(self._index, f'cortx{_DELIM}external{_DELIM}consul{_DELIM}endpoints')
+            ConfigManager.init("ha_setup", log_path=ha_log_path, level="INFO", skip_load=True)
+            # With new changes in GConf, consul http endpoint will be available at
+            # cortx>external>consul>endpoints[1] key
+            consul_endpoint = self._get_endpoints(GconfKeys.CONSUL_ENDPOINTS_NUM_KEY.value, \
+                                GconfKeys.CONSUL_ENDPOINT_KEY.value, const.consul_scheme)
             #========================================================#
             # consul Service endpoints from cluster.conf             #
             #____________________ cluster.conf ______________________#
             # endpoints:                                             #
             # - tcp://consul-server.default.svc.cluster.local:8301   #
             # - http://consul-server.default.svc.cluster.local:8500  #
             #========================================================#
-            # search for supported consul endpoint url from list of configured consul endpoints
-            filtered_consul_endpoints = list(filter(lambda x: isinstance(x, str) and urlparse(x).scheme == const.consul_scheme, consul_endpoints))
-            if not filtered_consul_endpoints:
+            if not consul_endpoint:
                 sys.stderr.write(f'Failed to get consul config. consul_config: {filtered_consul_endpoints}. \n')
```

Comment:
Check: Does this variable exist any more: **filtered_consul_endpoints** ?

Reply:
Thanks for pointing out this. Fixed now. Please check.

Reply:
Looks good now

Reply:
Deepak is not able to resolve the comments. I spoke with him and he is fine to close that. Hence closing on his behalf. Thanks!